tiatoolbox/wsicore/wsireader.py

```diff
@@ -1629,7 +1629,7 @@ def _find_params_from_baseline(self, location, baseline_read_size):
         baseline_size = np.array(self.info.slide_dimensions)
         image_size = np.array(self.img.shape[:2][::-1])
         size_ratio = image_size / baseline_size
-        image_location = np.array(location, dtype=np.float) * size_ratio
+        image_location = np.array(location, dtype=np.float32) * size_ratio
```

Comment:
To remove derprecation warning. Attached here
```
<stdin>:1: DeprecationWarning: `np.float` is a deprecated alias for the builtin `float`. To silence this warning, use `float` by itself. Doing this will not modify any behavior and is safe. If you specifically wanted the numpy scalar type, use `np.float64` here.
```